packages/toolpad-app/src/toolpad/propertyControls/SelectOptions.tsx

```diff
@@ -72,14 +80,20 @@ function SelectOptionsPropEditor({
 
   const handleOptionItemClick = React.useCallback(
     (index: number) => () => {
+      setOptionTextInputInvalid(false);
```

Comment:
if all this state does for now is check if the empty is empty, maybe it should be called `optionTextInputEmpty` instead?

Reply:
or what if we use a `optionTextInputError` state and you can set there any error message as a string?